funnel/extapi/boxoffice.py

```diff
@@ -21,7 +21,11 @@ def __init__(self, access_token, base_url=None):
 
     def get_orders(self, ic):
         url = urljoin(self.base_url, 'ic/{ic}/orders?access_token={token}'.format(ic=ic, token=self.access_token))
-        return requests.get(url).json().get('orders')
+        resp = requests.get(url)
+        if resp.status_code == 200:
+            return resp.json().get('orders')
+        else:
+            raise Exception(u"Invalid response from boxoffice server: {0!r}".format(resp))
```

Comment:
Why do you need this? The `.json()` call will raise an exception anyway.

Reply:
In general, I think raising `Exception` is lazy and indicates poor planning. There has to be a more suitable exception.